sale_automatic_workflow_job/models/automatic_workflow_job.py

```diff
@@ -45,21 +70,8 @@ def _validate_invoices(self, domain_filter):
             domain_filter
         )
 
-    def _do_validate_picking_job_options(self, picking, domain_filter):
-        description = _("Validate transfer {}").format(picking.display_name)
-        return {
-            "description": description,
-            "identity_key": identity_exact,
-        }
-
-    def _validate_pickings(self, domain_filter):
-        with_context = self.with_context(auto_delay_do_validation=True)
-        return super(AutomaticWorkflowJob, with_context)._validate_pickings(
-            domain_filter
-        )
-
     def _do_sale_done_job_options(self, sale, domain_filter):
-        description = _("Mark sales order {} as done").format(sale.display_name)
+        description = self.env._("Mark sales order %s as done", sale.display_name)
```

Comment:
```suggestion
        description = self.env._(f"Mark sales order {sale.display_name} as done")
```

Reply:
@vvrossem FTR using f strings will break translations because the string will be evaluated at run time making the string always different. Ideally we should have named placeholders like `_("Foo %(baz)s", baz=self.baz)`, but is not a blocker.

Reply:
Other reviewers suggested to use a positional argument if there is only one (as is the case here), and switch to named arguments in case of multiple. I like that one myself, but maybe that's all that it is, a preference?